Example/Kommunicate/LoginViewController.swift

```diff
@@ -0,0 +1,96 @@
+//
+//  LoginViewController.swift
+//  Kommunicate_Example
+//
+//  Created by Mukesh on 27/02/19.
+//  Copyright © 2019 CocoaPods. All rights reserved.
+//
+
+import UIKit
+import Kommunicate
+
+class LoginViewController: UIViewController {
+
+    @IBOutlet weak var userName: UITextField!
+    @IBOutlet weak var password: UITextField!
+
+    @IBOutlet weak var emailId: UITextField!
+
+    @IBOutlet weak var activityIndicator: UIActivityIndicatorView!
+    @IBOutlet weak var loginAsVisitorButton: UIButton!
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+
+        loginAsVisitorButton.layer.borderWidth = 1
+        loginAsVisitorButton.layer.borderColor = UIColor(hexString: "1588B2")?.cgColor
+    }
+
+    @IBAction func getStartedBtn(_ sender: AnyObject) {
+        let applicationId = AppDelegate.appId
+        setupApplicationKey(applicationId)
+
+        guard let userIdEntered = userName.text, !userIdEntered.isEmpty else {
+            let alertMessage = "Please enter a userId. If you are trying the app for the first time then just enter a random Id"
+            let alert = UIAlertController(
+                title: "Kommunicate login",
+                message: alertMessage,
+                preferredStyle: UIAlertController.Style.alert)
+            alert.addAction(UIAlertAction(title: "Okay", style: UIAlertAction.Style.default, handler: nil))
+            self.present(alert, animated: true, completion: nil)
+            return
+        }
+        let kmUser = userWithUserId(userIdEntered, andApplicationId: applicationId)
+
+        print("userName:: " , kmUser.userId)
+        if(!((emailId.text?.isEmpty)!)){
```

Comment:
Can we use let syntax or something else? Right now it kinda looks weird. Same with Password.
Also, Above print statement can be modified to print userId not userName.